src/main/java/com/amam/wizardschool/controller/StudentController.java

```diff
@@ -19,47 +20,49 @@ public StudentController(StudentService studentService) {
         this.studentService = studentService;
     }
 
-    @GetMapping("/all")
-    @Operation(summary = "Get all students")
-    public Collection<Student> getStudents() {
-        return studentService.getStudents();
+    @GetMapping
+    @Operation(summary = "Get all students / Get students by Age / Get students by Age between two values")
+    public ResponseEntity<Collection<Student>> getStudents(@RequestParam(required = false) Integer age,
+                                                           @RequestParam(required = false, defaultValue = "0") Integer minAge,
+                                                           @RequestParam(required = false, defaultValue = "1000") Integer maxAge) {
```

Comment:
```suggestion
                                                           @RequestParam(required = false, defaultValue = "0") int minAge,
                                                           @RequestParam(required = false, defaultValue = "1000") int maxAge) {
```
нет никакого смысла в использовании Integer, т.к. параметры not null из-за defaultValue